azure-pipelines.yml

```
@@ -60,7 +60,7 @@ extends:
       tsa:
         enabled: true
       binskim:
-        analyzeTargetGlob: $(Build.SourcesDirectory)/artifacts/bin/**.dll;$(Build.SourcesDirectory)/artifacts/bin/**.exe;
+        analyzeTargetGlob: +:f|**\*.dll;+:f|**\*.exe;
```

This isn't the easiest way to do this because the 1ES PT doesn't have a codepath for the easiest way. 😡 This code would need to be changed: https://dev.azure.com/dnceng/_git/1ESPipelineTemplates?path=/v1/Core/Steps/Windows.SDL.Binary.Analysis.yml&version=GBmain&line=610&lineEnd=652&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Anyway, it should be easier to use the regex to match this stuff. This one matches paths that have `artifacts\bin` in the path and end with either `.dll` or `.exe`. This is the suggested way to do this via the docs: https://dev.azure.com/securitytools/SecurityIntegration/_wiki/wikis/Guardian/1378/Glob-Format?anchor=optimization

I've also forced the value to be a string in YAML by wrapping it in single quotes. Just makes it easier for the YAML parser to know the data type because of the symbols and junk in there. It is just safer to do in this situation.

```suggestion
        analyzeTargetGlob: 'f:regex|artifacts\\bin\\.+\.(dll|exe)$'
```

---

Per offline chat, the above suggestion doesn't work because of bugs in 1ES:
##[error]AnalyzeArgumentNoValuesException: Argument Target has no values. Check your configuration. -- Additional arguments:Microsoft.Guardian.InvalidResponseFileContentsException: InvalidResponseFileContentsException: Cannot create a response file with zero arguments. Ensure that your arguments are correctly set up.